README.adoc

```diff
@@ -92,6 +92,7 @@ refute_nil(actual)
 
 `assert_equal` should always have expected value as first argument because if the assertion fails the
 error message would say expected "rubocop-minitest" received "rubocop" not the other way around.
+If you're used to working with RSpec then note this in the opposite order.
```

Comment:
I'd probably use the adminition syntax `NOTE:` here.

Reply:
Thanks, updated.